packages/ui/src/memberships/hooks/useMembers.ts

```diff
@@ -42,11 +42,17 @@ export const useMembers = ({ order, filter, perPage = 10 }: UseMemberProps) => {
 
 type FilterGqlInput = Pick<
   MembershipWhereInput,
-  'id_eq' | 'roles_some' | 'isVerified_eq' | 'isFoundingMember_eq' | 'handle_contains' | 'isCouncilMember_eq'
+  | 'id_eq'
+  | 'roles_some'
+  | 'isVerified_eq'
+  | 'isFoundingMember_eq'
+  | 'handle_contains'
+  | 'controllerAccount_eq'
+  | 'isCouncilMember_eq'
 >
 
 const filterToGqlInput = ({ search, roles, council, onlyVerified, onlyFounder }: MemberListFilter): FilterGqlInput => ({
-  ...(search ? { handle_contains: search } : {}),
+  ...(search ? { OR: [{ controllerAccount_eq: search }, { handle_contains: search }] } : {}),
```

Comment:
Please also include `rootAccount_eq`. Some users have same `rootAccount` and `controllerAccount`, but I saw many examples where users have them different. Thank you!